README.md

```diff
@@ -7,6 +7,43 @@ Carbon is a tool for loading data into [Symplectic Elements](https://support.sym
 
 Please refer to the [mitlib-tf-workloads-carbon](https://github.com/mitlibraries/mitlib-tf-workloads-carbon) for the deployment configuration.
 
+For more information on the Carbon application, please refer to our [internal documentation on Confluence](https://mitlibraries.atlassian.net/l/cp/1E6cMvuT).
+
+## Data flow
+
+This flowchart depicts the data flow from MIT's Data Warehouse to the application's in-memory buffered streams and to the final location of the output file, an XML file on the Elements FTP server.
+
+```mermaid
+---
+title:
+---
+
+flowchart TB
+   subgraph ext-source[Database]
+      mit-dwrhs[(MIT Data Warehouse)]
+   end
+   
+   subgraph in-memory[Application In-memory]
+      direction TB
+      rec-generator([Record generator])
```

Comment:
Is it accurate that this "Record generator" is a python generator from the data warehouse query results?  If so, I might suggest renaming to "Query Results Generator" to really drive home that it's looping over the records from the data warehouse.

Reply:
Yes! Update made!